packages/app-page-builder/src/editor/plugins/elements/paragraph/elementText.ts

```diff
@@ -20,14 +20,12 @@ const lexicalText = JSON.stringify({
                         version: 1
                     }
                 ],
-                styleId: "paragraph1",
-                name: "Paragraph 1",
                 direction: "ltr",
-                typographyStyles: {},
+                styles: [],
                 format: "",
                 indent: 0,
                 tag: "p",
-                type: "typography-el-node",
+                type: "base-paragraph-node",
```

Comment:
We are changing the `styles` and `type` here. Will this affect existing paragraphs added to a page before 5.35.1? Will this break anything?

Reply:
This will not affect existing nodes. This is the case when you drop a new paragraph node, so we will continue with the new node.